client/blocks/signup-form/social.jsx

```diff
@@ -81,7 +81,9 @@ class SocialSignupForm extends Component {
 
 	render() {
 		const uxMode = this.shouldUseRedirectFlow() ? 'redirect' : 'popup';
-		const redirectUri = uxMode === 'redirect' ? `https://${ window.location.host }/start` : null;
+		const host = typeof window !== 'undefined' && window.location.host;
```

Comment:
Most of these changes are duplicate logic from the file above. It would be really nice to extract it somehow 😕

Reply:
Agreed there is a some repetition here, we're not generating the same redirectUri though so I'm not sure it's worth it

Reply:
Ah, I missed that difference, thanks!